apps/src/StudioApp.js

```diff
@@ -39,6 +39,7 @@ import project from './code-studio/initApp/project';
 import * as assets from './code-studio/assets';
 import i18n from './code-studio/i18n';
 import AbuseError from './code-studio/components/abuse_error';
+import ObservableEvent from './ObservableEvent';
```

Comment:
I definitely prefer EventEmitter. Didn't realize we were already using that. I looked in `package.json` but didn't see the event emitter library.